src/Ev.ServiceBus/ServiceBusEngine.cs

```diff
@@ -12,38 +12,32 @@ namespace Ev.ServiceBus;
 
 public class ServiceBusEngine
 {
-    private readonly ILogger<ServiceBusEngine> _logger;
+    private readonly ILogger<LoggingExtensions.ServiceBusClientManagement> _serviceBusClientManagementLogger;
+    private readonly ILogger<LoggingExtensions.ServiceBusEngine> _serviceBusEngineLogger;
     private readonly IOptions<ServiceBusOptions> _options;
     private readonly ServiceBusRegistry _registry;
     private readonly MessageSenderFactory _messageSenderFactory;
     private readonly ReceiverWrapperFactory _receiverWrapperFactory;
 
     public ServiceBusEngine(
-        ILogger<ServiceBusEngine> logger,
         IOptions<ServiceBusOptions> options,
         ServiceBusRegistry registry,
         MessageSenderFactory messageSenderFactory,
-        ReceiverWrapperFactory receiverWrapperFactory)
+        ReceiverWrapperFactory receiverWrapperFactory, 
+        ILogger<LoggingExtensions.ServiceBusClientManagement> serviceBusClientManagementLogger,
+        ILogger<LoggingExtensions.ServiceBusEngine> serviceBusEngineLogger)
     {
-        _logger = logger;
         _options = options;
         _registry = registry;
         _messageSenderFactory = messageSenderFactory;
         _receiverWrapperFactory = receiverWrapperFactory;
+        _serviceBusClientManagementLogger = serviceBusClientManagementLogger;
+        _serviceBusEngineLogger = serviceBusEngineLogger;
     }
 
     public async Task StartAll()
     {
-        _logger.LogInformation("[Ev.ServiceBus] Starting azure service bus clients");
-
-        if (_options.Value.Settings.Enabled == false)
-        {
-            _logger.LogInformation("[Ev.ServiceBus] Reception and dispatch of messages have been deactivated through configuration.");
-        }
-        if (_options.Value.Settings.Enabled && _options.Value.Settings.ReceiveMessages == false)
-        {
-            _logger.LogInformation("[Ev.ServiceBus] Reception of messages have been deactivated through configuration.");
```

Comment:
Please keep this log. 
it is only produced during initialization of the app, so the cost of having them is low.
And they are useful to know exactly which resource are initialized